packages/studio-ui/src/web/components/Layout/Toolbar/index.tsx

```diff
@@ -51,10 +54,33 @@ const Toolbar: FC<Props> = (props) => {
           ) : (
             <Fragment>
               <Tooltip content={<div>{lang.tr('toolbar.help')}</div>}>
-                <button id="statusbar_tutorial" className={style.item} onClick={toggleGuidedTour}>
-                  <Icon color="#1a1e22" icon="help" iconSize={16} />
-                  <span className={style.label}>{lang.tr('toolbar.tutorial')}</span>
-                </button>
+                <Popover
+                  content={
+                    <Menu>
+                      <MenuItem icon="learning" text={lang.tr('toolbar.tutorial')} onClick={toggleGuidedTour} />
+                      <MenuItem icon="people" text={lang.tr('toolbar.forum')} onClick={() => window.open(FORUM_LINK)} />
+                      <MenuItem icon="book" text={lang.tr('toolbar.docs')} onClick={() => window.open(DOCS_LINK)} />
+                    </Menu>
+                  }
+                  minimal
+                  isOpen={isHelpPopoverOpen}
+                  position={Position.TOP_RIGHT}
+                  canEscapeKeyClose
+                  onClose={() => setHelpPopoverOpen(false)}
+                  fill
+                  modifiers={{
+                    preventOverflow: { enabled: true, boundariesElement: 'window' }
+                  }}
+                >
+                  <button
+                    id="statusbar_tutorial"
+                    className={style.item}
+                    onClick={() => setHelpPopoverOpen(!isHelpPopoverOpen)}
+                  >
+                    <Icon color="#1a1e22" icon="help" iconSize={16} />
+                    <span className={style.label}>{lang.tr('toolbar.help')}</span>
+                  </button>
+                </Popover>
```

Comment:
pass this as target and remove the click handler you'll get an uncontrolled popover 🎉